share/spice/just_delete_me/just_delete_me.js

```diff
@@ -13,7 +13,9 @@
             return Spice.failed('just_delete_me');
         }
         var matchPartial = function(item) {
-            return (item.name.toLowerCase().includes(this) || item.domains.some(domain => domain.includes(this)));
+            return (item.name.toLowerCase().includes(this) || item.domains.some(function(domain) {
+                return domain.includes(this)
```

Comment:
@riqpe yes I've just realize this -- can you please go ahead and make a PR? I'm a little busy now